### PR TITLE
Update HomePage

### PR DESCRIPTION
**Pull request by CarlosMilano**
# What does this PR do?

Update the HomePage

# Where should the reviewer start?

app/routes/index.tsx

# How should this be manually tested?

# Any background context you want to provide?

# What are the relevant tickets?

#274 

# Screenshots

Captura de pantalla 2023-11-14 a la(s) 6 11 50 p m

Captura de pantalla 2023-11-14 a la(s) 6 12 18 p m

Captura de pantalla 2023-11-14 a la(s) 6 12 35 p m

# Questions

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Review by beatriz-sanchez-wizeline — COMMENTED** [review]
Wow 😃  I like the new look and feel! Much better than before! 👍 

**Functionality/Design comments:**
- Once logged in there is no quick way to go to the "Dashboard", that is, the Projects view. I would suggest adding this at the top where the login buttons were when not signed in.  
- On mobile, the applicant log in and wizeline log in that are supposed to be at the top are lost and there is no menu.
  - The wizeline log in is only at the top and when on mobile it is not available at the bottom so no way to log in at all. Not sure why only applicant log in is at the bottom and the wizeline log in is not.  
- Adding a container to the sections below the cover pic might make the design look more modern.
- Slightly changing the background color of each section below the cover pic, by only a few tones might make their separation clearer and cleaner to read.
- Contact/About links are broken, pages don't exist. Maybe we can comment them out while they are not functional.
- Our personal experience has no sample data so I would suggest commenting out the section until we have something to put there or to just make up some examples that we can update with the aid of the marketing team.
- Because most of the items in the nav bar are or will be external links, I feel they are more suited for a footer or a section at the bottom than a nav bar. I would keep the black bar only and possibly move the red wizeline logo there. 

**Some philosophical questions:**
At first sight it was not clear to me the purpose of this page. 
- As an applicant: I am wasn't sure what is the difference between "https://www.wizeline.com/offerings/" (link at the top under "work") and Wizelabs was. I was missing something like "Are you a student? Apply to work on a industry job as an intern"
- As a wizeliner: I felt this page was not meant for me as it was referring me to external links like "job offerings (work)" and in the "what is wizeline?" section mentioned a recruitment process which I already did. I was missing something like "Are you a wizeliner? Apply to one of our internal projects and acquire new skills!" 

With this in mind, I think that changing the title of the "What is wizelabs?" section to something along the lines of "What you'll get if you join" and adding a section at the top explaining "What is wizelabs? (for the two kinds of users: an external applicant and for a wizeliner)" might make it more clear.

**Closing comments**
Overall, I do like the new look on web and mobile :) I think there is room for improvement but it is much better than before. Do let me know your thoughts.

**Comment by martinrobled0** [comment]
Can you add some screen shots of the mobile version?

**Comment by CarlosMilano** [comment]
I made the menu responsive and changed the things that Bety told me, I just want to know your opinion about the mobile menu, speaking of the design, how would you like me to make the mobile design?

**Comment by AxelUZ** [comment]
> I made the menu responsive and changed the things that Bety told me, I just want to know your opinion about the mobile menu, speaking of the design, how would you like me to make the mobile design?

The design looks good, maybe we could make the menu in mobile similar to this one [https://www.wizeline.com/](url)
But we could keep the logo in the menu, I like the way it looks.

Captura de pantalla 2023-11-22 a la(s) 1 47 37 a m

**Comment by beatriz-sanchez-wizeline** [comment]
> I made the menu responsive and changed the things that Bety told me

Thanks @CarlosMilano , I can see that the landing page looks better now.

> I just want to know your opinion about the mobile menu, speaking of the design, how would you like me to make the mobile design?

I think the mobile menu is looking good. The design that @AxelUZ is suggesting looks nice and we might want to get there at some point but the current one is functional enough for me. I think leaving the sign-in options at the bottom (as it is now) was a good choice. **My only concern is that the "projects" link is not there** and I think this is a very important link to have on the menu, specially if logged in. **Same thing for the desktop version**, the projects link must be at the top, either in the nav bar or where the "wizeline sign in" button was.

Another thing I noticed @CarlosMilano  is that, there is no **"WizeLABS"** anywhere on the page, the main title is **"WizeLINE"** is that expected?

**Comment by martinrobled0** [comment]
Looks nice but I think adding a hover effect to the menu items would enhance the overall aesthetic. Thinking in the user experience.

image

**Comment by martinrobled0** [comment]
> Looks nice but I think adding a _hover_ effect to the menu items would enhance the overall aesthetic. Thinking in the user experience.
> 
> image

the hover effect looks good now, but Is it possible to make the hover effect the size of the div and change the cursor icon?

File: app/routes/index.tsx [hunk 1]
```diff
@@ -39,24 +64,82 @@ export default function Index() {
...
           src="/wizeletters.png"
           alt="Wizeline letters for homepage header"
           width={190}
         />
       </HomeHeader>
+      <IconButton>
+        {menu ? (
+          <CloseIcon onClick={handleMenu} style={{ fontSize: "35px" }} />
+        ) : (
+          <DehazeIcon onClick={handleMenu} style={{ fontSize: "35px" }} />
```
Comment (martinrobled0) [inline app/routes/index.tsx:79]: why Dehaze?

File: app/routes/index.styles.tsx [hunk 2]
```diff
@@ -10,11 +13,110 @@ export const HomePageContainer = styled.div`
...
+  font-family: "Roboto", sans-serif;
+  font-size: 1.5rem;
+  color: rgb(175, 46, 51);
+  background-color: transparent;
+  border: none;
+  transition: all 0.3s ease-in-out;
+  width: 100%;
+  height: 80px;
+
+  &:hover {
```
Comment (martinrobled0) [inline app/routes/index.styles.tsx:102]: you're. missing the `cursor: pointer;`

**Review by martinrobled0 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**